src/seedu/addressbook/commands/ListUsersCommand.java

```diff
@@ -0,0 +1,39 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.List;
+
+public class ListUsersCommand extends Command{
+
+    public static final String COMMAND_WORD = "list_users";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays all created users and password.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String SUCCESS_MESSAGE = "Users and password listed.";
+
+    @Override
+    public CommandResult execute() {
+
+        try {
+            File users = new File("users.txt");
+
```

Comment:
Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.